Packs/PingIdentity/Integrations/PingOne/PingOne.yml

```diff
@@ -0,0 +1,48 @@
+category: Identity and Access Management
+commonfields:
+  id: PingOne
+  version: -1
+configuration:
+- display: Your PingOne Environment ID
+  additionalinfo: "Environment ID is available under the PingOne Dashboard->Environment Properties"
+  name: environment_id
+  required: true
+  type: 4
+- display: PingOne Region
+  additionalinfo: "PingOne has 3 regions, US, EU and Asia"
+  name: region
+  required: true
+  type: 4
+- display: Trust any certificate (not secure)
+  name: insecure
+  required: false
+  type: 8
+- display: Use system proxy settings
+  name: proxy
+  required: false
+  type: 8
+description: 'Integrates with the PingOne Management API to retrieve, create and delete users'
+display: PingOne
+name: PingOne
+script:
+  commands:
+  - arguments:
+    - default: true
+      description: 'Username to unlock'
+      isArray: false
+      name: username
+      required: true
+      secret: false
+    deprecated: false
+    description: "Unlock a user's account"
+    execution: true
+    name: pingone-unlock-user
+  isfetch: false
+  runonce: false
+  script: '-'
+  type: python
+  subtype: python3
+  dockerimage: demisto/python3:3.9.5.21272
+fromversion: 5.5.0
```

Comment:
Why is the from version is 5.5.0 and not a newer one?

Reply:
```suggestion
fromversion: 6.0.0
```

Reply:
My apologies the first commit did not include all changes, not sure what exactly happen.   Should be version 6.0.0 now.